# Downstream patch to skip a test failing on OBS.
# tests: skip tests/rm/ext3-perf.sh temporarily as it hangs on OBS
# (but otherwise is known to work locally).
---
 tests/rm/ext3-perf.sh |    5 +++++
 1 file changed, 5 insertions(+)

Index: tests/rm/ext3-perf.sh
===================================================================
--- tests/rm/ext3-perf.sh.orig
+++ tests/rm/ext3-perf.sh
@@ -19,6 +19,11 @@
 . "${srcdir=.}/tests/init.sh"; path_prepend_ ./src
 print_ver_ rm
 
+# Skip this test: it's known to work locally (and it was working for a long
+# time on OBS), but it started to fail in Sep 2021 on OBS - seemingly on all
+# platforms (i586, x84_64, ppc64le, ppc64, aarch64, s390x).
+skip_ "SUSE: disabled for now as known to be failing on OBS"
+
 very_expensive_
 
 # In a circa 2008 benchmark, using rm -rf to remove a 400k-entry directory took:
